ivy_tests/test_ivy/test_functional/test_experimental/test_core/test_linalg.py

```diff
@@ -490,6 +490,60 @@ def test_multi_dot(dtype_x, test_flags, backend_fw, fn_name):
     )
 
 
+@handle_test(
+    fn_tree="functional.ivy.experimental.dot",
+    dtype_xy=helpers.dtype_and_values(
+        available_dtypes=(ivy.float32, ivy.float64),
+        num_arrays=2,
+        large_abs_safety_factor=8,
+        small_abs_safety_factor=8,
+        safety_factor_scale="log",
+        min_num_dims=1,
+        max_num_dims=1,
+    ),
+    ground_truth_backend="numpy",
```

Comment:
I think this is throwing an error

Reply:
Ah, I thought it made sense to use numpy as the ground truth since most of the backend implementations rely on it in the end.